test/e2e/serial/tests/workload_unschedulable.go

```diff
@@ -676,8 +676,23 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload unsched
 			err = fxt.Client.Create(context.TODO(), dp)
 			Expect(err).ToNot(HaveOccurred())
 
-			By(fmt.Sprintf("checking deployment pods have been scheduled with the topology aware scheduler %q ", schedulerName))
+			// although the deployment pods will be pending thus the deployment will not be counted as complete,
```

Comment:
this is to catch existing flakes or is it a prevention mechanism?

Reply:
its goal is to prevent having flakiness in failures in upcoming checks

Reply:
ack, good.